test/plugins/test/autolink.js

```diff
@@ -107,18 +107,17 @@ describe('AutoLink', function() {
 		assert.equal(data, '<p><span>@test</span> </p>');
 	});
 
-	it('should not create a link if last word is not a valid URL', function() {
+	it('should to try to create a link from the last word', function() {
```

Comment:
Grammar seems a bit off here ("should to try to"), and I think our linter is going to complain about [starting an `it` with should](https://github.com/liferay/eslint-config-liferay/blob/master/plugins/eslint-plugin-liferay/docs/rules/no-it-should.md).

How about just:

```js
it('creates a link when the last word is a URL', function() {
```

Reply:
Much better thanks